tests/layer_tests/ovc_python_api_tests/test_pytorch.py

```diff
@@ -747,12 +747,10 @@ def forward(self, x):
     param1 = ov.opset10.parameter(shape, dtype=np.float32)
     weights = ov.opset10.constant(net.weights.numpy(force=True))
     cast1 = ov.opset10.convert(weights, np.float32)
-    sub1_const = np.float16(0.5).reshape(1, 1, 1, 1)
-    mul1_const = np.float16(0.02).reshape(1, 1, 1, 1)
-    sub1_const_decompress = ov.opset10.convert(sub1_const, np.float32)
-    mul1_const_decompress = ov.opset10.convert(mul1_const, np.float32)
-    sub1 = ov.opset10.subtract(cast1, sub1_const_decompress)
-    mul1 = ov.opset10.multiply(sub1, mul1_const_decompress)
+    sub1_const = np.float32(0.5).reshape(1, 1, 1, 1)
+    mul1_const = np.float32(0.02).reshape(1, 1, 1, 1)
+    sub1 = ov.opset10.subtract(cast1, sub1_const)
+    mul1 = ov.opset10.multiply(sub1, mul1_const)
```

Comment:
It doesn't looks as enough test coverage for the feature. For example, no tests for FakeConvert and optional parts of ZP.